addons/sale/models/sale_order_line.py

```diff
@@ -1315,9 +1315,6 @@ def _convert_to_sol_currency(self, amount, currency):
             )
         return amount
 
-    def has_valued_move_ids(self):
-        return self.move_ids
-
```

Comment:
Please don't remove and/or change function signatures in stable versions. A third party module may be using it.

In this case, could you make it return True since `move_ids` is a sale_stock field, then call super() in `sale_stock/has_valued_move_ids` ?